dask/bag/tests/test_bag.py

```diff
@@ -1190,7 +1190,9 @@ def test_zip(npartitions, hi=1000):
     evens = db.from_sequence(range(0, hi, 2), npartitions=npartitions)
     odds = db.from_sequence(range(1, hi, 2), npartitions=npartitions)
     pairs = db.zip(evens, odds)
-    assert pairs.npartitions == npartitions
```

Comment:
This assertion was only true because the length of the bag happened to be a multiple of 100. Updated it to be a little more robust.